Kwartracker/Scenes/Alert/AlertView.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  AlertView.swift
+//  Kwartracker
+//
+//  Created by Veverly Mae Ricaza Veverly on 4/27/21.
+//
+
+import SwiftUI
+
+struct AlertView: View {
+    var body: some View {
+        ZStack {
+            Color.gray.opacity(0.4).edgesIgnoringSafeArea(.all)
+            MainAlertView()
+                .frame(width: 220, height: 240, alignment: .center)
```

Comment:
If the alert is uniform in size, move the magic numbers to a constant, and I think the height should be dynamic not fixed